Javascript/README.md

```diff
@@ -49,6 +49,23 @@ if (time < 60) {
 }
 ```
 
+There is an objective reason for this. If you write the following code
+
+```javascript
+function myFunction()
+{
+	return
+	{
+		property1: 'value1',
+		property2: 'value2'
+	};
+}
+```
+
+`return\n` will be interpreted as `return;\n` and your function will be broken. Thus, for consistency matter, we want
+any curly braces to be on the same line as whatever they open.
```

Comment:
This is two different type of braces (object and block).
the following code will have the same problem, this is unrelated to brace:
```javascript
function myFunction()
{
	return
	'value1';
}
```